edk2toolext/invocables/edk2_setup.py

```diff
@@ -130,35 +149,37 @@ def Go(self):
 
         # Grab the remaining Git repos.
         if required_repos:
+
             # Git Repos: STEP 1 --------------------------------------
             # Make sure that the repos are all synced.
-            try:
-                edk2_logging.log_progress("## Syncing Git repositories: %s..." % ", ".join(required_repos))
-                RunCmd("git", 'submodule sync -- ' + " ".join(required_repos),
-                       workingdir=workspace_path, logging_level=logging.DEBUG, raise_exception_on_nonzero=True)
+            for required_repo in required_repos:
+                try:
+                    edk2_logging.log_progress(f"## Syncing Git repositories: {required_repo.path}...")
+                    RunCmd("git", f'submodule sync -- {required_repo.path}',
```

Comment:
Will this be a perf hit? I think there's a large startup cost of invoking git. You can use a python map expression to create the list of needed for the old code. [x.path for x in required_repos]

Reply:
ok